src/runtime/composables/useKbd.ts

```diff
@@ -1,13 +1,20 @@
 import { createSharedComposable } from '@vueuse/core'
 import { ref, computed, onMounted } from 'vue'
 
+type KbdSpecificMap = {
+  meta: string
+  alt: string
+  ctrl: string
+}
+
 export const kbdKeysMap = {
   meta: '',
+  ctrl: '',
+  alt: '',
+  win: '⊞',
   command: '⌘',
   shift: '⇧',
-  ctrl: '⌃',
   option: '⌥',
```

Comment:
Why keep option but not ctrl here?

Reply:
`ctrl` depends on the system where the user is currently located. But if they want to display `option` code regardless of the system, they can do it